story.csv

```diff
@@ -0,0 +1 @@
+this is new story has to be added once created PR request
```

Comment:
can you added name of the story

Reply:
why this all has to be done
as soon as adding functionality start working  on instagram real functionality